src/main/java/sirius/biz/i5/Transformer.java

```diff
@@ -152,15 +154,25 @@ private int transform(Entry<Field, Transform> e, Object object, byte[] data, int
                                                     field.getName())
                             .handle();
         } catch (Exception ex) {
-            throw Exceptions.handle()
-                            .to(I5Connector.LOG)
-                            .error(ex)
-                            .withSystemErrorMessage("Error while transforming '%s.%s' to %s: %s (%s)",
-                                                    object.getClass().getName(),
-                                                    field.getName(),
-                                                    info.targetType().getName())
-                            .handle();
+            if (info.ignoreErrors()) {
+                I5Connector.LOG.FINE("Ignoring a conversion error for %s of %s: %s (%s)",
```

Comment:
Add Exceptions.ignore so the Exception can be retrieved later for debugging purpose